legal-api/report-templates/template-parts/common/benefitCompanyStmt.html

```diff
@@ -0,0 +1,34 @@
+<div class="no-page-break">
+    {% if business.legalType == 'BEN' or alteration.business.legalType == 'BEN' %}
+        <div class="section-title mt-4">Benefit Company Statement</div>
+        <div class="section-data mt-4">
+            This company is a benefit company and, as such, has purposes that include conducting its
+            business in a responsible and sustainable manner and promoting one or more public benefits.
+        </div>
+    {% endif %}
+</div>
+
+{% if header.reportType != 'alterationNotice' %}
+    <div class="no-page-break">
+        <div class="section-data mt-4">
+            <span class="bold">Name of Company:</span>
+            {{business.legalName}}
+        </div>
+    </div>
+
+    <div class="no-page-break">
+        {% if nameTranslations|length > 0 %}
```

Comment:
I think this should be in its own common section too

Reply:
Do you mean the translation section in benefitCompanyStmt.html? We have already a common component for translations called nameTranslation.html, but it has different format. Then, I kept separately, do you want me to include some conditions in the file to have both translations in the same and depending on the use to change the format?

Reply:
This is a sample of ShareStructure in alteration notice.
[alterationNotice_shareStructure.pdf](https://github.com/bcgov/lear/files/6126569/alterationNotice_shareStructure.pdf)